### PR TITLE
Updated is_funky in curated records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1605.

Updates all `mz:is_funky` property values to 1 in records that we've previously curated through a neighbourhood update issue. Does not require PIP work.

### CODE REVIEW

File: data/102/061/079/102061079.geojson [hunk 1]
```diff
@@ -15,7 +17,8 @@
     "mps:latitude":40.679066,
     "mps:longitude":-73.990374,
     "mz:hierarchy_label":1,
-    "mz:is_funky":1,
+    "mz:is_current":-1,
+    "mz:is_funky":0,
```
Comment (nvkelso) [inline data/102/061/079/102061079.geojson]: This one should stay `funky`.

**Comment by nvkelso** [comment]
There seem to be a lot of records in this PR, please check to see if it can be scoped down farther.

**Comment by stepps00** [comment]
I was overly cautious with the first commit. An upcoming commit will revert these changes and only operate on curated polygons (not from Quattroshapes, Yerbashapes, Zetashapes, etc) that are not flagged with `is_current` = 0.

**Comment by stepps00** [comment]
Closing in preference of a new branch,

**Pull request closed without merging**